Source/WebCore/Modules/fetch/FetchHeaders.cpp

```diff
@@ -181,13 +207,33 @@ ExceptionOr<String> FetchHeaders::get(const String& name) const
 {
     if (!isValidHTTPToken(name))
         return Exception { TypeError, makeString("Invalid header name: '", name, "'") };
+
+    if (equalIgnoringASCIICase(name, "set-cookie"_s)) {
+        if (m_setCookieValues.isEmpty())
+            return String();
+        StringBuilder builder;
+        for (const auto& value : m_setCookieValues) {
+            if (!builder.isEmpty())
+                builder.append(", ");
+            builder.append(value);
+        }
+        return builder.toString();
+    }
     return m_headers.get(name);
 }
 
+Vector<String> FetchHeaders::getSetCookie() const
```

Comment:
Should probably be const Vector<String>&

Reply:
Fixed.